### PR TITLE
Update CDAP Upgrade docs

### PR DESCRIPTION
**Pull request by sreevatsanraman**

### CODE REVIEW

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 1]
```diff
@@ -618,13 +618,19 @@ you have a use case for it, please reach out to us at `cdap-user@googlegroups.co
...
 
    - Using APT (on one line)::
 
        $ sudo apt-get install cdap cdap-gateway \
              cdap-hbase-compat-0.94 cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
-             cdap-kafka cdap-master cdap-security cdap-web-app
+             cdap-kafka cdap-master cdap-security cdap-ui
+
+#  Remove link to older cdap-web-app service
+       $ rm -rf /etc/init.d/cdap-web-app
```
Comment (dereklwood) [inline cdap-docs/admin-manual/source/installation/installation.rst]: this should not be necessary

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 2]
```diff
@@ -618,13 +618,19 @@ you have a use case for it, please reach out to us at `cdap-user@googlegroups.co
...
+
+   We have deprecated the cdap-web-app package. So the links to older cdap-web-app service
+   should be removed
```
Comment (dereklwood) [inline cdap-docs/admin-manual/source/installation/installation.rst]: I had the following:

Note that the ``cdap-ui`` package is new in CDAP 3.0.0, and replaces the ``cdap-web-app`` package.
On some Ubuntu systems, the upgrade may leave the ``cdap-web-app`` package in an ``ic`` state, meaning only its
configuration files are installed.  It is safe to completely remove this package, using ``apt-get purge cdap-web-app``

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 3]
```diff
@@ -638,17 +638,18 @@ you have a use case for it, please reach out to us at `cdap-user@googlegroups.co
...
 
      $ for i in `ls /etc/init.d/ | grep cdap` ; do sudo service $i start ; done
      
+#. Run the Flow Queue pending metrics corrector::
+
+     $ /opt/cdap/master/bin/svc-master run co.cask.cdap.data.tools.flow.FlowQueuePendingCorrector
+
+   This will correct the pending metrics for flows. This is a new metric that was introduced in 
+   CDAP 3.0; flows that existed before the upgrade to 3.0 do not have a correct value for this
+   metric and running the tool provides a one time correction.
```
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/installation.rst]: one-time

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 4]
```diff
@@ -638,17 +638,18 @@ you have a use case for it, please reach out to us at `cdap-user@googlegroups.co
...
+
 #. This will allow you to see your old run history, logs, and |---| if you migrated your 
    old metrics with the *Data Migration Tool* |---| metrics.
```
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/installation.rst]: This line needs editing because you removed this tool on the lines above.

**Comment by dereklwood** [comment]
There are still a couple references to Webapp: lines 28 and 70.  otherwise, LGTM

**Comment by awholegunch** [comment]
Most important is to remove or adjust the reference to the removed Data Migration Tool.

File: cdap-docs/admin-manual/source/installation/installation.rst (hunk 3 above)
Comment (sreevatsanraman) [inline cdap-docs/admin-manual/source/installation/installation.rst]: Fixed.

File: cdap-docs/admin-manual/source/installation/installation.rst (hunk 4 above)
Comment (sreevatsanraman) [inline cdap-docs/admin-manual/source/installation/installation.rst]: Fixed.

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 5]
```diff
@@ -618,13 +616,15 @@ you have a use case for it, please reach out to us at `cdap-user@googlegroups.co
...
+             cdap-kafka cdap-master cdap-security cdap-ui
 
    - Using APT (on one line)::
 
        $ sudo apt-get install cdap cdap-gateway \
              cdap-hbase-compat-0.94 cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
-             cdap-kafka cdap-master cdap-security cdap-web-app
+             cdap-kafka cdap-master cdap-security cdap-ui
+
+       **Note:** We have deprecated the cdap-web-app package in favor of cdap-ui package
```
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/installation.rst]: This line should be brought out to left to match the hyphen, as it applies to both Yum and APT. Also, it is not a literal command...

**Comment by awholegunch** [comment]
There are other references to "cdap-web-app" in that doc that should be removed. Lines 316 & 320.

**Comment by awholegunch** [comment]
Line 71 references **CDAP Webapp** -> CDAP UI...and line 29.

Some of these changes (line 29) are also made in a pending PR...

**Comment by awholegunch** [comment]
LGTM.

**Pull request merged**